wit/streams.wit

```diff
@@ -85,92 +85,107 @@ interface streams {
     /// `skip` will always report end-of-stream rather than producing more
     /// data.
     ///
-    /// This function returns the number of bytes skipped, along with a
-    /// `stream-status` indicating whether the end of the stream was
-    /// reached. The returned value will be at most `len`; it may be less.
+    /// This function returns the number of bytes skipped, along with a bool
+    /// indicating whether the end of the stream was reached. The returned
+    /// value will be at most `len`; it may be less.
```

Comment:
```suggestion
    /// This function returns the number of bytes skipped, along with a
    /// `stream-status` indicating whether the end of the stream was
    /// reached. The returned value will be at most `len`; it may be less.
```

I think you've accidentally reverted some changes from #36

Reply:
Thank you for catching this!